client/web/src/App.jsx

```diff
@@ -47,6 +47,7 @@ const App = () => {
   const shouldPlayAudio = useRef(false);
   const audioQueue = useRef([]);
   const isConnected = useRef(false);
+  const isMobile = window.innerWidth <= 768; 
```

Comment:
this technically isn't "isMobile". There is specific mobile detection libs like https://www.npmjs.com/package/is-mobile

Given you do want to detect the screen size, maybe change to isNarrowScreen